jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -161,7 +166,8 @@ pipeline {
                                 inputManifest: "manifests/${INPUT_MANIFEST}",
                                 platform: 'linux',
                                 architecture: 'x64',
-                                distribution: 'tar'
+                                distribution: 'tar',
+                                continueOnError: ${params.CONTINUE_ON_ERROR}
```

Comment:
I think you either just use params.<> or wrap double quotes around it?
I could be mistaken here tho.

Reply:
Assuming test both true and false on staging jenkins already?

Reply:
Yes tested on both. Works as expected!

Reply:
You were right! It needs to be either of you mentioned. Fixed here https://github.com/opensearch-project/opensearch-build/pull/3888